compiler/parser/parser.cpp

```diff
@@ -412,7 +412,7 @@ static void ensureRequiredStandardModulesAreParsed() {
       UnresolvedSymExpr* oldModNameExpr = toUnresolvedSymExpr(moduleExpr);
 
       if (oldModNameExpr == NULL) {
-        INT_FATAL("It seems an internal module is using a mod.submod form");
+        continue;
```

Comment:
I should point out that Michael also tries to do this in #15257 and I would really like *someone* to fix the whole function XD

Reply:
It passed testing, so I'm not convinced this approach is broken until someone proves it is...  IIRC, this is also some pretty fiddly code to make sure that a user doesn't accidentally hijack our copy of ChapelBase.chpl (say) without our noticing and that there are existing tests to ensure it's working...